.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
```

Comment:
I deleted dependabot from the other repos because I think it will only create noise we don't pay attention to. Could be interesting to think through holistically adding dependabot or something similar for these github gems, but I'd be inclined punt on that and to delete this for now